src/printers/jsx.js

```diff
@@ -77,4 +77,25 @@ export default {
   JSXText: ({node}) => node.value.trim()
     ? Tokens.string(node.value.trim())
     : null,
+
+  JSXMemberExpression: ({node, print}) => [
+    print(node.object),
+    Tokens.string('.'),
+    print(node.property),
+  ],
+
+  JSXNamespacedName: ({node, print}) => [
+    print(node.namespace),
+    Tokens.string(':'),
```

Comment:
`Tokens.colon()`

Not sure why exactly, but for most punctuation I have helpers for that token specifically